### ISSUE TITLE
Replace Buffer with Uint8Array

### ISSUE THREAD

**Issue by xcaptain** [ton-org/ton-core#49]
I'm developing a dapp using sveltekit + tonconnectui, I have to install the `vite-plugin-node-polyfills` package because `@ton/core`  uses `Buffer` which is a nodejs only type. I hope you can replace it with `Uint8Array` so we can better use it in browser.

**Comment by xcaptain** [comment]
@dvlkv can you take a look at it? do you have any plan to  remove nodejs dependency?

**Comment by dvlkv** [comment]
Yes we do have plans removing Buffer in the next major version. However, I cannot provide you any estimates on this update.

**Comment by xcaptain** [comment]
This PR shows it's not hard to replace Buffer with Uint8Array. I'm happy to help if you need.

**Comment by pxbuffer** [comment]
@dvlkv Hi! Perhaps you could estimate now?
